src/main/java/se/knowit/bookitevent/model/EventValidator.java

```diff
@@ -0,0 +1,53 @@
+package se.knowit.bookitevent.model;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class EventValidator {
+    
+    private static final String RSVP_ERROR_TEMPLATE = "RVSP of Event at '%s' is set after start of Event at '%s";
+    private static final String END_TIME_ERROR_TEMPLATE = "End of Event at '%s' is set before start of Event at '%s";
+    
+    public Event ensureEventIsValidOrThrowException(Event incomingEvent) {
```

Comment:
I think it is more correct to make this method static, since it doesn't need to be bound to an instance of the class.

Reply:
True. It could be static. On the other hand it would make it harder to test if we want to inject different validators at some point.

Reply:
I'm a bit vary of using static methods. Difficult to mock them out unless we use PowerMock or something.